js/react_native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java

```diff
@@ -46,6 +48,13 @@ public class OnnxruntimeModule extends ReactContextBaseJavaModule {
   private static OrtEnvironment ortEnvironment = OrtEnvironment.getEnvironment();
   private static Map<String, OrtSession> sessionMap = new HashMap<>();
 
+  private static BigInteger nextSessionId = new BigInteger("0");
+  private static String getNextSessionKey() {
+    String key = nextSessionId.toString(2);
+    nextSessionId.add(new BigInteger("1"));
```

Comment:
```suggestion
    nextSessionId = nextSessionId.add(BigInteger.valueOf(1));
```
need to assign it as add() doesn't update the value in place
https://stackoverflow.com/a/1783929

Reply:
Good point. updated.